src/config/cron.md

```diff
@@ -1,13 +1,28 @@
 # Cron
 
-Void Linux comes without a default cron daemon, you can choose one of multiple
-cron implementations, by installing the package and enabling the system service.
+[cron](https://en.wikipedia.org/wiki/Cron) is a daemon for running programs at
+regular intervals. The programs and intervals are specified by using
+[crontab(1)](https://man.voidlinux.org/crontab.1) to edit a `crontab` file.
+`crontab` files can be system-wide or user-specific; to edit the system crontab,
+run `crontab` as the superuser:
 
-Available choices include [cronie](https://github.com/cronie-crond/cronie/),
+```
+# crontab -e
+```
```

Comment:
"it is necessary to run `crontab -e` as the superuser", avoids the code block and I prefer the structure. What do you think?

Reply:
Sure, i've done the same thing in a cron guide i've started. :-) Will change.